tests/test_notifications.py

```diff
@@ -1,1174 +0,0 @@
-import collections
```

Comment:
So tests have been moved to a different location? Are there any changes in the file that needs to be ported to wherever we've ported this one to?